src/main/java/com/microsoft/sqlserver/jdbc/SQLServerParameterMetaData.java

```diff
@@ -929,6 +958,10 @@ public boolean isSigned(int param) throws SQLServerException {
 
     String getTVPSchemaFromStoredProcedure(int param) throws SQLServerException {
         checkClosed();
-        return (String) getParameterInfo(param).get("SS_TYPE_SCHEMA_NAME");
+        Map<String, Object> info = getParameterInfo(param);
+        if (null != info) {
+            return (String) info.get("SS_TYPE_SCHEMA_NAME");
+        }
+        return null;
```

Comment:
return `null` or `""`?

Reply:
should return null imo

Reply:
It's as per existing design.. but I guess we should throw exception if no metadata is available for specified `param`.